### ISSUE TITLE
Clicks not in game cause deck tracker window to move

### ISSUE THREAD

**Issue by olinWong** [mtgatracker/mtgatracker#421]
On 4.5.4 
Whenever i click on anything not in the game window, it causes the MTGA deck tracker window to move left or diagonally left & up.

Good case: Behavior not shown when clicking inside the game window.
Affected use case: When say, using a widescreen & multitasking with the game in a smaller window alongside another window: 
Clicks outside the game window cause this behavior, to the point of it going off screen and disappearing.

**Comment by shawkinsl** [comment]
@WizardRubic reported this at one point as well. Maybe we can find some similarity in y'all setups? Can you tell us:

- Windows version (win7, win10, 32 bit or or x64)
- Screen resolution (if multiple screens please list all)
- Graphics card(s)

**Comment by olinWong** [comment]
Windows 10, x64
1440x3440 Widescreen (1x)
AMD RX580, CPU has no iGPU

**Comment by WizardRubic** [comment]
Windows 10, x64
2 monitors, 1920x1080, 2560x1080
Intel HD Graphics 630, GeForce GTX 1050

(My setup looks real similar to @shawkinsl 's setup apart from different second monitor, maybe we have the same laptop? :o )

I tried to fix this bug before in software but could only find work around. There was something about calling browserWindow.setBounds() moving the screen even if the bounds didn't change and I couldn't figure out why. There was a workaround I found but it's much less than ideal(set bounds width/height to very high values and pin to 0,0/manually adjust this whenever someone tried to drag).

The bug doesn't happen if I never move the tracker window from the default position once it's started up.

**Comment by WizardRubic** [comment]
[https://youtu.be/7aXxgCGgomE](https://youtu.be/7aXxgCGgomE)

Well I put a video up about the lines of code I changed to make it work on my machine :p It'll for sure break it for every other user except me and olinWong but it shows why I think it might be a rounding error. I think when setBounds is called it does some sorta calculation to determine where the window should end up before calling the Windows api. I'm not 100% sure though. I'll look into it more later!

**Comment by olinWong** [comment]
@WizardRubic The tracker still moves even with the game closed.
Since it still happens with the game closed, it might be linked to the path that minimizes/maximizes the tracker depending on if you do a L or R click.
Ex: Left click-->maximize tracker trigger-->redraw (changes due to environment?)

Also had a issue on tracker redraw on decklists populating the tracker, but not with card populating the tracker when playing games.
Assumptions: Upper left corner of tracker probably 0,0 & is probably redrawn from that point downwards.
Observed: Drifts to the left or diagonally left & up. 
In reference to itself, it wouldn't round down lower than 0, I think.
Q: Is clicking outside the game window recognized as a out of bounds click?
It's possible in reference to the entire screen that clicking outside the game might mean something that makes the redraw shift negatively.

PS: Also got something obscuring the very bottom of my tracker in game. Should do pictures and check if this a dupe.

**Comment by WizardRubic** [comment]
> Since it still happens with the game closed, it might be linked to the path that minimizes/maximizes the tracker depending on if you do a L or R click.

Yeah that will call our resizeWindow() function too! So it is the same issue, I'm gonna see if there's an official way to get a build out with the changes I made so you can test it. 

I remember something about there being a way to have appveyor create builds of branches :p
